doc/sdk-cli-reference.md

```diff
@@ -88,9 +88,17 @@ operator-sdk completion bash
 # ex: ts=4 sw=4 et filetype=sh
 ```
 
+## print-deps
+
+Prints the most recent Golang packages and versions required by operators. Prints in columnar format by default.
+
+### Flags
+
+* `--as-file` Print packages and versions in Gopkg.toml format.
+
```

Comment:
@estroz Sorry for the late review but it might be good to have a truncated example output here for a particular version, as we do for other commands.
We can show the `--as-file` output.